memberships/templates/memberships/login.html

```diff
@@ -0,0 +1,15 @@
+{% extends "base.html" %}
+
+{% block content %}
+  <h2>Member Login</h2>
+  <form action="/memberships/login/" method="POST">
```

Comment:
Are we able to use the url helper here? Like in `register.html`

Reply:
Ohhhhhh yeeeeaaaa.... ...done...